### ISSUE TITLE
Wrong polarization occurs along dendrities

### ISSUE THREAD

**Issue by schmitts** [arbor-sim/arbor#1533]
The attached script shows wrong polarization along a dendritic tree. The cell features passive mechanisms only and is stimulated excitatorily at the soma. With further distance to the soma, the depolarization turns wrongfully into a polarization.

Arbor @ c57be8a20fcb171615f7ea9f447615344d371b0e.
[wrong_polarization_head.pdf](https://github.com/arbor-sim/arbor/files/6508448/wrong_polarization_head.pdf)
[wrong_polarization_head.py.gz](https://github.com/arbor-sim/arbor/files/6508449/wrong_polarization_head.py.gz)

**Comment by schmitts** [comment]
The wrong polarization can also be observed in the attached example invoked like so:
![single_cell_cable_result](https://user-images.githubusercontent.com/1502412/120166561-fddd7300-c1fc-11eb-9e4b-4ae9c1bd6594.png)

```console
single_cell_cable.py --length 1000 --cv_policy_max_extent 500
```

**Comment by halfflat** [comment]
The problem is ultimately one of interpolation: voltage probes at a point will attempt to interpolate the membrane voltage at that point based on the geometry and electrical properties. When there is only a small number of CVs on a cable, this interpolation can be misleading.

I made a version of the passive example you have above, which compares the output from point voltage probes, cell-wide voltage probes (which report a voltage per CV cable), and the exact solution. Geometry and electrical properties are taken from the Rallpack1 test.

Source: [cable_vary_cv.py.gz](https://github.com/arbor-sim/arbor/files/6603291/cable_vary_cv.py.gz)

Example run, with command line as follows, describing a run with 30 voltage probes along the cable, measuring at times 2 ms, 5 ms, and 10 ms, with 2 CVs, 5 CVs and 10 CVs. The interpolation here is fairly dull, because the cable has constant radius, resistivity, etc.
```
cable_vary_cv.py -p 30 -t 2 -t 5 -t 10 -c 2 -c 5 -c 10 -o varying.svg
```
![varying](https://user-images.githubusercontent.com/556816/120906587-0994c380-c65b-11eb-8d69-1e76c4f6eee8.png)

Even with just two CVs, I don't think this behaviour is a bug; at the very least, it is operating according to our design, which might be a bad design :) If there's interest, I could tidy up the script and add it to the examples.

**Comment by halfflat** [comment]
I'm going to close this with: surprising, but not a bug. We can look at fixes in #1580 !
